pkg/apis/core/validation/validation.go

```diff
@@ -3773,35 +3775,54 @@ func podSpecHasContainer(spec *core.PodSpec, containerName string) bool {
 }
 
 const (
+	// a sysctl name regex
+	SysctlFmt string = "(" + SysctlSegmentFmt + "\\.)*" + SysctlSegmentFmt
+
+	// a sysctl name regex contains a slash
+	SysctlRelaxedFmt string = "(" + SysctlSegmentContainSlashFmt + "\\.)*" + SysctlSegmentContainSlashFmt
+
 	// a sysctl segment regex, concatenated with dots to form a sysctl name
 	SysctlSegmentFmt string = "[a-z0-9]([-_a-z0-9]*[a-z0-9])?"
 
-	// a sysctl name regex
-	SysctlFmt string = "(" + SysctlSegmentFmt + "\\.)*" + SysctlSegmentFmt
+	// a sysctl segment regex contains a slash, concatenated with dots to form a sysctl name
+	SysctlSegmentContainSlashFmt string = "[a-z0-9]([-_/a-z0-9]*[a-z0-9])?"
```

Comment:
this allows multiple slashes in a row... is that correct?

Reply:
Explain the same as above, need to modify the regex.